src/client/studies/TCGA/MedianDistribution.js

```diff
@@ -28,7 +27,7 @@ class MedianDistribution extends React.Component {
 
   bellCurveType = 'median'
 
-  verticals = false
+  verticals = 0
```

Comment:
is this numerical way of dealing with verticals okay?

Reply:
I think this is how it should be done in `IntronAnalysis` as well. `verticalsCount` might be a better name though.